tests/Tests/PHPUnit/Traits/AdditionalAssertionsTraitTest.php

```diff
@@ -133,7 +134,8 @@ public function __toString()
      */
     protected function classUsedTraitFactory()
     {
-        return new class extends \PHPUnit\Framework\TestCase {
+        return new class extends \PHPUnit\Framework\TestCase
```

Comment:
`{` in anonimous classes shold be on same line with class definition (CS rule) [2]

Reply:
Done